features/site_configuration.feature

```diff
@@ -161,8 +161,8 @@ Feature: Site configuration
       And I have a post layout that contains "Post Layout: {{ content }} built at {{ page.date | date_to_xmlschema }}"
       And I have an "index.html" page with layout "page" that contains "site index page"
       And I have a configuration file with:
-        | key         | value            |
-        | timezone    | America/New_York |
+        | key         | value        |
+        | timezone    | UTC+04:00    |
```

Comment:
This should be UTC-4 if it is meant to be New York but you notice how the posts have their own offset so they are unaffected by this:

```text
| entry2    | 2013-04-10 03:14 -0400 | post    | content for entry2. |
```

becomes: 

```text
Post Layout: <p>content for entry2.</p>\n built at 2013-04-10T03:14:00-04:00
```

That seems to invalidate this test. I believe we'd need to remove the timezone offsets in the `date` segment of the posts.